contents/ios/IntroToIos.md

```diff
@@ -0,0 +1,103 @@
+- Next, you should register for a free Apple Developer Account. Don't confuse this with the paid iOS Developer Program! Anyone can register for the Apple Developer Account for free. Go to [Apple's Developer Website](https://developer.apple.com/register/) to do this. It requires you to have an Apple ID, so you can use your existing one if you already have an Apple ID.
+- Once you have an Apple Developer Account, you can either directly download Xcode from the [website](https://developer.apple.com/xcode/), or search for Xcode on your Mac App Store. Xcode is the Integrated Development Environment (IDE) that provides you with everything you need to develop an iOS app from scratch. It also somes with the iPhone and iPad simulator that you will need to test your application.
+
+Native iOS applications can be written in [Objective-C or Swift](https://android.jlelse.eu/objective-c-or-swift-which-technology-to-learn-for-ios-app-development-3c681d1a05ac). Swift is a relatively newer language, introduced only in 2014, while Objective-C is more of an old school programming language. If you are just starting out on iOS development, you should strongly consider using Swift. The main reason being that many of the documentations and help on the internet are written for iOS development in Swift, hence it will be easier to look for resources that target Swift instead of Objective-C. Furthermore, Apple mostly regards Swift as the main language for iOS programming now, and Objective-C support is now more of a backward-compatability. 
+
+If you are new to Swift, you may want to read up on our Swift article: [Introduction to Swift]({{baseUrl}}/contents/swift/welcome-to-swift.html)
+
+If you are ready to begin developing your first iOS application, you will want to check out these iOS development tutorials and articles meant for first-timers:
+- [Codewithchris: How to make an iPhone app](https://codewithchris.com/how-to-make-an-iphone-app/)
+- [raywenderlich.com "How to build a simple iOS app"](https://www.raywenderlich.com/3114-ios-tutorial-how-to-create-a-simple-iphone-app-part-1-3)
+
+If you have worked with iOS applications before, are familiar with the whole development process and would like to take the next step, check out these more in-depth tutorials for iOS development and its intracacies:
+- [MVC Design Pattern for iOS apps](https://www.raywenderlich.com/1073-model-view-controller-mvc-in-ios-a-modern-approach)
+- [Auto Layout](https://www.raywenderlich.com/443-auto-layout-tutorial-in-ios-11-getting-started)
+- [iOS Animations](https://www.raywenderlich.com/363-ios-animation-tutorial-getting-started)
+
+## 6. Useful links and further reading
```

Comment:
No need for section numbers (to be consistent with other articles). Number 5 is missing anyway.